src/components/app-top-bar.js

```diff
@@ -242,7 +242,8 @@ const AppTopBar = ({ user, tabIndex, onChangeTab, userManager }) => {
                 theme={themeLocal}
                 onEquipmentLabellingClick={handleChangeUseName}
                 equipmentLabelling={useNameLocal}
-                withElementsSearch={Boolean(studyUuid)}
+                withElementsSearch={true}
```

Comment:
why this change ?

Reply:
as the search function can now be disabled it prevent it from disappearing